tests/testthat/test-crunch-data-frame.R

```diff
@@ -0,0 +1,184 @@
+context("CrunchDataFrame Methods")
```

Comment:
Throughout this file (and everywhere), please put a space after a comma

Reply:
It looks like there are a bunch of places where we need to fix this throughout the tests. Rather than send a massive change the includes a bunch of white space, I'll add this as an issue to go through and fix separately elsewhere. I've fixed the files I'm working on here.